src/UITests/GettingStartedPage.cs

```diff
@@ -9,29 +9,33 @@ public class GettingStartedPage : AIWinSession
     {
         [TestMethod]
         [TestCategory("NoStrongName")]
-        public void VerifyGettingStartedTitle()
+        public void GettingStartedPageTests()
+        {
+            VerifyGettingStartedTitle();
+            VerifyAccessibility();
+        }
+
+        private void VerifyGettingStartedTitle()
         {
             Assert.AreEqual("Accessibility Insights for Windows - ", driver.Title);
         }
 
-        [TestMethod]
-        [TestCategory("NoStrongName")]
-        public void VerifyAccessibility()
+        private void VerifyAccessibility()
         {
-            var result = driver.ScanAIWin();
+            var result = driver.ScanAIWin(TestContext.ResultsDirectory);
             Assert.AreEqual(0, result.errors);
         }
 
-        [ClassInitialize]
-        public static void ClassInitialize(TestContext context)
+        [TestInitialize]
+        public void TestInitialize()
         {
-            Setup(context);
+            Setup();
```

Comment:
When the checks pass it would be great to see if the video includes navigation. If not, I wonder if it matters enough to us to put navigation inside the main test method instead

Reply:
Agreed, let's change this if it doesn't show up in the video.